### ISSUE TITLE
print cost estimate

### ISSUE THREAD

**Issue by casperdcl** [iterative/terraform-provider-iterative#564]
Can we use APIs to retrieve an estimated hourly price for `machine` on each provider to print to the screen on `apply`? atm it only says `{machine} on {cloud}`. Would be great to append `(~${price}/h)`

Also on `refresh`/`destroy` we could calculate total cost estimate by multiplying by uptime.

If we implement this we could then even auto-select `cloud` based on lowest price :P

**Comment by 0x2b3bfa0** [comment]
### Related

* https://github.com/iterative/terraform-provider-iterative/issues/158#issuecomment-934826522
* https://github.com/iterative/terraform-provider-iterative/issues/158#issuecomment-1108684522

**Comment by 0x2b3bfa0** [comment]
Not sure of the value of this feature until we narrow down the target audience.

**Comment by 0x2b3bfa0** [comment]
> _auto-select `cloud` based on lowest price_

Yes, sure, and even `machine` from a range of tolerated specifications. 🙃 Still, this doesn't look like something a measly client–side tool should be doing.

Switching cloud providers [potentially] implies having credentials for all of them and paying for data transfer costs.

**Comment by 0x2b3bfa0** [comment]
target audience: any user wondering how much something costs without reaching for a calculator.

> _To me it's closer to p1 than p2._

Using priority numbers as the reference framework might not be adequate in this occasion. 🤔 I'd say that `issues < this < issues < planning < design` without fear to being mistaken.
